core-requirements.txt

```diff
@@ -1,8 +1,8 @@
-numpy>=1.16.4
-pandas>=0.25.0
+numpy>=1.19.1
+pandas>=1.1.0
 scipy>=1.2.1
-scikit-learn>=0.23
-scikit-optimize>=0.8
+scikit-learn>=0.23.1
```

Comment:
@angela97lin so the problem was specifically in sklearn 0.23.0?

Reply:
> @angela97lin so the problem was specifically in sklearn 0.23.0?

Combination of scikit-learn 0.23 and scikit-optimize 0.8 and numpy: https://github.com/scikit-optimize/scikit-optimize/issues/902